draft-sharabayko-mops-srt.md

```diff
@@ -1156,58 +1315,61 @@ never become aware. The missing packet problem is resolved this way:
-or larger packet burst drop. Packets received after their "play time"
-are dropped if Too-Late Packet Drop feature is enabled
+The goal of the SRT Timestamp Based Packet Delivery (TSBPD) mechanism is to reproduce 
+the output of the sending application (e.g., encoder) at the input of the receiving 
+application (e.g., decoder) in live data transmission mode (see {{data-transmission-mode}}). 
+It attempts to reproduce the timing of packets committed by the sending application to 
+the SRT sender. This allows packets to be scheduled for delivery by the SRT receiver, 
+making them ready to be read by the receiving application (see {{fig-latency-points}}).
+
+The SRT receiver, using the timestamp of the SRT data packet header, delivers packets to 
+a receiving application with a fixed minimum delay from the time the packet was scheduled 
+for sending on the SRT sender side. Basically, the sender timestamp in the received packet
+is adjusted to the receiver’s local time (compensating for the time drift or different 
+time zones) before releasing the packet to the application. Packets can be withheld by 
+the SRT receiver for a configured receiver delay. A higher delay can accommodate a larger 
+uniform packet drop rate, or a larger packet burst drop. Packets received after their 
+"play time" are dropped if the Too-Late Packet Drop feature is enabled 
 (see {{too-late-packet-drop}}).
 
-The packet timestamp (in microseconds) is relative to the SRT connection creation time.
+The packet timestamp (in microseconds) is relative to the SRT connection creation time. 
+Packets are inserted based on the sequence number in the header field. The origin time 
+(in microseconds) of the packet is already sampled when a packet is first submitted by 
+the application to the SRT sender. The TSBPD feature uses this time to stamp the packet 
+for first transmission and any subsequent retransmission. This timestamp and the 
+configured SRT latency ({{srt-latency}}) control the recovery buffer size and the 
+instant that packets are delivered at the destination (the aforementioned "play time" 
+which is decided by adding the timestamp to the configured latency).
+
 It is worth mentioning that the use of the packet sending time to stamp the packets is
-inappropriate for TSBPD feature since a new time (current sending time) is used for retransmitted packets,
-putting them out of order when inserted at their proper place in the stream. Packets are
-inserted based on the sequence number in the header field. The origin time (in microseconds)
-of the packet is already sampled when a packet is first submitted by the application to the SRT sender.
-The TSBPD feature uses this time to stamp the packet for first transmission and any subsequent retransmission.
-This timestamp and the configured SRT latency ({{srt-latency}}) control the recovery buffer size and the instant that packets
-are delivered at the destination.
```

Comment:
This part starting from "Packets are inserted .." is deleted. Was it an intent?

Reply:
Will replace